source/include/FreeRTOS_IP.h

```diff
@@ -413,9 +417,7 @@ BaseType_t xIsNetworkDownEventPending( void );
  * Socket has had activity, reset the timer so it will not be closed
  * because of inactivity
  */
-#if ( ( ipconfigHAS_DEBUG_PRINTF != 0 ) || ( ipconfigHAS_PRINTF != 0 ) )
```

Comment:
Is it necessary to remove this condition check?
If so, should we remove the same one in FreeRTOS_TCP_State_Handling.c for the function implementation?

Reply:
Not all of the uses of FreeRTOS_GetTCPStateName were guarded by the condition check, so it's either remove the check from the declaration or fill in the remaining usage checks. This was also causing test failures in FreeRTOS_TCP_State_Handling_utest.c so some of the configFiles for testing would have to be updated.